### PR TITLE
Update gitkraken instructions & screenshots

### PR DESCRIPTION
**Pull request by sjspielman**
Closes https://github.com/AlexsLemonade/training-admin/issues/1156

This PR updates GitKraken instructions. A couple notes:
- I updated the GitKraken website screenshots for Mac because I could, but not for Windows because I can't. So, I removed the Windows screenshot altogether. At that point, I wondered if we should just remove the Mac screenshots too - seems like an overkill item to have to maintain... Thoughts?
- I tested this with a fresh _everything_: Fresh GitKraken install (and more importantly a `rm -rf ~/.gitkraken`), and a brand spanking new GitHub account. Through this, I learned that GitKraken no longer prompts you to agree to their free license during setup, so I went ahead and removed this screenshot.
- I did some recreational revisions to replace `git` with Git throughout the setup docs

Reviewer note: If you want to serve this while reviewing, you'll need to navigate directly to the setup pages since we currently don't have them linked on the OpenRRP-flavor of the website. So, you'll want `local host address/setup_instructions/mac_installation_instructions.html`, etc.

### CODE REVIEW

File: docs/setup_instructions/screenshots/github/gitkraken_git_signin.png
Comment (jashapiro) [inline docs/setup_instructions/screenshots/github/gitkraken_git_signin.png:1]: This also seems not needed, though here I can expect a bit more of a reason to keep it to tell people they got to the right place.

**Review by jashapiro — COMMENTED** [review]
This looks good to me, but on the topic of removing screenshots, I think the main ones to get rid of may be the website ones? I expect people will be able to find these buttons.

So I would remove a couple images, which will involve a bit more text change, which is why I am not just approving straight away.

File: docs/setup_instructions/screenshots/mac/gitkraken_mac_1.png
[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/setup_instructions/screenshots/mac/gitkraken_mac_1.png:1]: yup, that was these!

>  At that point, I wondered if we should just remove the Mac screenshots too - seems like an overkill item to have to maintain

File: docs/setup_instructions/mac_installation_instructions.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ title: macOS Installation Instructions
 
 ## Apple Command Line Tools
```
Comment (jashapiro) [inline docs/setup_instructions/mac_installation_instructions.md:12]: Just noting from another PR that we sometimes call these "Developer tools" but Command Line Tools is better so we should try to be consistent

**Review by jashapiro — APPROVED** [review]
LGTM

**Pull request merged**